libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -204,12 +204,14 @@ ap_object AP_RangeFinder_Backend method orientation Rotation'enum
 ap_object AP_RangeFinder_Backend method type uint8_t
 ap_object AP_RangeFinder_Backend method status uint8_t
 ap_object AP_RangeFinder_Backend method handle_script_msg boolean float'skip_check
+ap_object AP_RangeFinder_Backend method handle_script_msg_ex boolean float'skip_check float'skip_check
 
 singleton RangeFinder depends (!defined(HAL_BUILD_AP_PERIPH) || defined(HAL_PERIPH_ENABLE_RANGEFINDER))
 singleton RangeFinder rename rangefinder
 singleton RangeFinder method num_sensors uint8_t
 singleton RangeFinder method has_orientation boolean Rotation'enum ROTATION_NONE ROTATION_MAX-1
-singleton RangeFinder method distance_cm_orient uint16_t Rotation'enum ROTATION_NONE ROTATION_MAX-1
+singleton RangeFinder method distance_cm_orient_ex void Rotation'enum ROTATION_NONE ROTATION_MAX-1 uint16_t'Ref int8_t'Ref
+singleton RangeFinder method distance_cm_orient_ex rename distance_cm_orient
```

Comment:
Looks like distance_cm_orient used to return 1 item, and now returns 2. Does this work w/ old scripts that expect 1 item?

Reply:
Yes. Similar to the way function arguments are handled, lua lines up what the caller wants and what the function returns and fills in nils or drops values. So, the returned signal_quality value gets dropped if a caller doesn't pick it up. A little bit inefficient but if avoids adding a new interface method.